api/notd/api/models_v1.py

```diff
@@ -68,6 +67,8 @@ def from_model(cls, model: UiData):
         )
 
 class ApiRegistryToken(BaseModel):
```

Comment:
the lines below should be gone after the merge but you should be adding transactionCount to the UiData class above